tests/check/check_h26xsigned_auth.c

```diff
@@ -1828,18 +1829,23 @@ generate_and_set_private_key_on_camera_side(struct sv_setting setting,
   // Add an I-NALU to trigger a SEI.
   sv_rc = signed_video_add_nalu_for_signing(sv, i_nalu->data, i_nalu->data_size);
   ck_assert_int_eq(sv_rc, SV_OK);
-  sv_rc = signed_video_get_nalu_to_prepend(sv, &nalu_to_prepend);
+
+  size_t sei_size = 0;
+  sv_rc = signed_video_get_sei(sv, NULL, &sei_size);
+  ck_assert(sei_size > 0);
   ck_assert_int_eq(sv_rc, SV_OK);
-  *sei = nalu_list_create_item(
-      nalu_to_prepend.nalu_data, nalu_to_prepend.nalu_data_size, setting.codec);
-  sv_rc = signed_video_get_nalu_to_prepend(sv, &nalu_to_prepend);
+  uint8_t *nalu_data = malloc(sei_size);
+  sv_rc = signed_video_get_sei(sv, nalu_data, &sei_size);
+  ck_assert_int_eq(sv_rc, SV_OK);
+  *sei = nalu_list_create_item(nalu_data, sei_size, setting.codec);
+  sv_rc = signed_video_get_sei(sv, nalu_data, &sei_size);
```

Comment:
This is an incorrect operation. You should pass in `NULL` instead of `nalu_data`. Otherwise you could by accident overwrite the data in the just created item.